### ISSUE TITLE
Integrate "Lifeboat" (native mobile app) with Where@ location server

### ISSUE THREAD

**Issue by fabacab** [betterangels/buoy#73]
There is another free software project called [Where@](https://whereat.io/) that has a functional Android app for their own server; we should use [this code](https://github.com/whereat/whereat-android) as the foundation for the "Lifeboat" mobile app for Buoy and build it to work with the WP-API backend that Buoy currently uses. However, it should not be changed such that it no longer works with the Where@ server, either.

In other words: Lifeboat should support BOTH a Where@ location server and a Buoy server.

**Comment by fabacab** [comment]
I've sent an email to the Where@ project to see if they're interested in helping with this.
